src/main/java/org/zalando/nakadi/config/NakadiSettings.java

```diff
@@ -74,4 +74,8 @@ public long getEventMaxBytes() {
         return eventMaxBytes;
     }
 
+    public long getTimelineWaitTimeoutMs() {
```

Comment:
This probably will be replaced by https://github.com/zalando/nakadi/blob/master/src/main/java/org/zalando/nakadi/config/NakadiSettings.java#L80 ?